qa-tests-backend/scripts/lib.sh

```diff
@@ -0,0 +1,23 @@
+deploy_default_psp() {
+    info "Deploy Default PSP for stackrox namespace"
+    "${ROOT}/scripts/ci/create-default-psp.sh"
+}
+
+deploy_webhook_server() {
+    info "Deploy Webhook server"
+
+    local certs_dir
+    certs_dir="$(mktemp -d)"
+    "${ROOT}/scripts/ci/create-webhookserver.sh" "${certs_dir}"
+    ci_export GENERIC_WEBHOOK_SERVER_CA_CONTENTS "$(cat "${certs_dir}/ca.crt")"
+}
+
+get_ECR_docker_pull_password() {
+    info "Get AWS ECR Docker Pull Password"
+
+    aws --version
+    local pass
+    pass="$(aws --region="${AWS_ECR_REGISTRY_REGION}" ecr get-login-password)"
+    ci_export AWS_ECR_DOCKER_PULL_PASSWORD "${pass}"
```

Comment:
Do we need to consider the case where get-login-password fails, what's the behavior of ci_export when the password is wrong / empty / an error message?

Reply:
This was a refactor, I pulled out the code from the `qa-tests-backend/scripts/run-part-1.sh`. This seems to be working for the longest time so I wouldn't touch it in this PR. But its a fair question.